test/cloffeine/core_test.clj

```diff
@@ -232,3 +233,18 @@
       (is (= 2 @reloads))
       (is (= 43 @(async-loading-cache/get alcache :key))))))
   
+(deftest compute
+  (let [c (cache/make-cache)
+        k "key"
+        remapper (fn [_k v]
+                   (if (some? v)
+                     (str v "bar")
+                     "foo"))]
+    (cache/put! c k "foo")
+    (is (= "foobar" (cache/compute c k remapper)))
+    (doall 
```

Comment:
I think this is an overkill.
If it does once, twice, then I believe it will do it 100 times.
I wouldn't test it.